### DISCUSSION TITLE
Be able to import a font

### DISCUSSION THREAD

**Discussion by LeCodeEtMoi** [ScratchAddons/ScratchAddons#5324] [Ideas]
**Is it possible to create an "add font" button in the font bar?**
This will allow you to customize these Scratch projects and give them a personality. Also it would be a time saver for those who create their font on Scratch. 
If anyone is interested in creating this addon, please let me know.
Thanks :)

**Comment by CST1229** [comment]
I don't think *importing* fonts is possible, but fonts that are installed on the user's computer are definitely possible.
